docs/source/contents/4_draft_mapping.rst

```diff
@@ -1331,6 +1331,297 @@ location
 recordInfo
 ==========
 
+recordIdentifier
+----------------
+
+Use Case
+^^^^^^^^
+
+Unremediated records often contain identifiers for the record. These take a couple of different forms. The Heilman
+collection and Volunteer Voices collections contain this element. In Volunteer Voices the identifier is simply the adminDB
+value with 'record' appended to the beginning (`e.g. volvoices:2352 <https://digital.lib.utk.edu/collections/islandora/object/volvoices%3A2352/datastream/MODS/view>`_).
+
+Justification
+^^^^^^^^^^^^^
+
+As the basic root of the <recordIdentifier> value is already present in the identifier element in all cases and the
+<recordIdentifier> value is never used on its own, there is no reason to retain these values.
+
+Xpath
+^^^^^
+
+"mods:recordInfo" / "mods:recordIdentifier"
+
+Decision
+^^^^^^^^
+
+All <recordIdentifier> values should be dropped, so no RDF example is included below.
+
+Here's an `example record - heilman:1001 <https://digital.lib.utk.edu/collections/islandora/object/heilman%3A1001/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <identifier type="local">Pseudolarix_0858</identifier>
+    <recordInfo>
+        <recordIdentifier>record_Pseudolarix_0858</recordIdentifier>
+    </recordInfo>
+
+languageOfCataloging
+--------------------
+
+Use Case
+^^^^^^^^
+
+All of the recently migrated SCOUT to TEI collections (e.g. American Civil War Collection, Tennessee Documentary History, etc.)
+as well as some of UTK's less recent collections (e.g. Sanborn, mpabaker, etc.) contain the element languageOfCataloging.
+In total, it is found in approximately 6,000 records. Note that in all cases the language is English, but this information
+is represented as both a code ('eng') and a text value ('English').
+
+Justification
+^^^^^^^^^^^^^
+
+Currently <languageOfCataloging> is not publicly displayed anywhere outside of the MODS XML. The values of this element
+do have the potential to be used if UTK has materials that might warrant cataloging in another language, but currently
+this is not the case. An example of a project that includes two records, one catalogued in Spanish and one in English, is
+`UNC's New Roots / Nuevas Raíces <https://newroots.lib.unc.edu/>`_. While UTK may want to pursue a project like this in the
+future, presently it seems unlikely that it will. More importantly, if such a project became a priority, it would not be
+difficult to distinguish via code UTK's existing English records from records in another language. If we did want to create
+a project like this, information on the language of cataloging could be added across the repository with minimal effort.
+
+Xpath
+^^^^^
+
+"mods:recordInfo" / "mods:languageOfCataloging"
```

Comment:
Normally, we express an xpath here.  Are you trying to express:

```
mods:recordInfo/mods:languageOfCataloging
```

Reply:
This was resolved in https://github.com/UTKcataloging/mods_to_rdf/pull/27/commits/2bd47b57b297b1805e1b9a7e6f517ce517eda056